src/Maker/AdminMaker.php

```diff
@@ -38,7 +38,7 @@ final class AdminMaker extends AbstractMaker
      */
     private $projectDirectory;
     /**
-     * @var string[]
+     * @var array<string, ModelManagerInterface|null>
```

Comment:
When it will be null?

Reply:
It's set in the constructor indeed.

we could also write 
```
private $availableModelManagers = [];
```

Reply:
This is forced by static analysis because of this line

```php
$this->modelManager = $this->availableModelManagers[$this->managerType] ?: current($this->availableModelManagers);
    }
```

Reply:
Maybe if you use ?? instead of ?: is better?

Reply:
yes, perfect!